### DISCUSSION TITLE
I can't get student pack while having a valid student account (not expired but soon)

### DISCUSSION THREAD

**Discussion by hw88hw88** [community/community#51602] [GitHub Education]
After logging in my GitHub account, I should be able to receive the benefits of the student developer pack.
However, GitHub failed to verify my student status. My student status is still valid and not expired yet.

Please help!

**Comment by ramazancetinkaya** [comment]
@hw88hw88, please kindly raise a ticket to [GitHub Support](https://support.github.com/) for the support engineers to investigate the issue.

They usually respond pretty quickly (within a couple of working days).

Hope this clarifies.

If you find this reply useful, please kindly 'Mark as answer' so that others know this issue is already addressed.

Have a good day.

**Reply by hw88hw88** [reply]
We can't open tickets for issues of GitHub Education.

Ref:
[https://share.internxt.com/d/sh/file/e7523fd23ef209c48d72/affff06d4a875f8084cfb641da47b379bac677c37e245c459e409d55f7b4e25f](url)
